tests/unit/test_validate_batch_operator.py

```diff
@@ -222,29 +218,3 @@ def test_context_type_cloud(self, mocker: MockerFixture):
 
         # assert
         mock_gx.get_context.assert_called_once_with(mode=context_type)
-
-    def test_batch_parameters(self):
```

Comment:
Isn't this still valid?

Reply:
the behavior under test here was replaced by the tests in `test_common`. If we keep it, this test basically becomes a check that we're calling into `run_validation` correctly. Which might be a good idea - happy to add it if you'd like

Reply:
Non-blocking, but I'm definitely more interested in tests around the public API, i.e. the operators.